transform/mattermost-analytics/models/marts/web_app/platform/_platform__models.yml

```diff
@@ -11,71 +11,48 @@ models:
-      - name: context_app_namespace          
       - name: id
         tests:
           - not_null
           - unique
-      - name: user_actual_id                
-      - name: context_library_name           
-      - name: type                           
-      - name: context_app_version            
-      - name: user_actual_role               
-      - name: context_page_url               
-      - name: context_os_name                
-      - name: context_page_title             
-      - name: anonymous_id                   
-      - name: context_app_build              
-      - name: context_page_search            
-      - name: context_library_version        
-      - name: context_ip                     
-      - name: context_useragent              
-      - name: context_app_name               
-      - name: context_locale                 
-      - name: context_screen_density         
-      - name: context_page_path              
-      - name: context_os_version             
-      - name: category                       
-      - name: duration                     
-      - name: num_of_request                 
-      - name: max_api_resource_size          
-      - name: longest_api_resource_duration  
-      - name: user_id                        
-      - name: root_id                        
-      - name: post_id                        
-      - name: sort                           
-      - name: team_id                        
-      - name: version                        
-      - name: keyword                        
-      - name: count                          
-      - name: gfyid                          
-      - name: field                          
-      - name: plugin_id                      
-      - name: installed_version              
-      - name: group_constrained              
-      - name: value                          
-      - name: include_deleted                
-      - name: role                           
-      - name: privacy                        
-      - name: scheme_id                      
-      - name: metric                         
-      - name: error                          
-      - name: num_invitations_sent           
-      - name: num_invitations                
-      - name: channel_sidebar                
-      - name: app                            
-      - name: method                         
-      - name: remaining                      
-      - name: screen                         
-      - name: filter                         
-      - name: version                        
-      - name: original_timestamp             
-      - name: sent_at                        
-      - name: uuid_ts                        
-      - name: received_at
-      - name: event_date
+      -name: anonymous_id
+    	-name: received_at
+    	-name: sent_at
+    	-name: original_timestamp
+    	-name: timestamp
+    	-name: context_ip
+    	-name: event
+    	-name: event_text
```

Comment:
`event` and `event_text` look to be new columns - are they needed in looker? (did we miss them in the initial iteration)?

Reply:
They aren't in Looker, they are the default rudderstack columns, we get these from the macro https://github.com/mattermost/mattermost-data-warehouse/blob/master/transform/mattermost-analytics/macros/rudderstack/get_rudderstack_columns.sql

Reply:
Wondering what value do we get by getting the `event` column for performance events. The only value it contains is `event`.

Reply:
its only 1 value `event` from this column

Reply:
yes..

Reply:
We'll scrub them in the next iterations, now that we've removed the `null`